### PR TITLE
updating scripts to support cascade deletion on policy groups

### PR DESCRIPTION
**Pull request by lahiru823**

### CODE REVIEW

File: modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js [hunk 1]
```diff
@@ -205,11 +205,11 @@ var module = function () {
                 return utility.assertEqual(adapter, predicate);
             }, null);
 
-            log.info('Locating saved asset: ' + stringify(artifact) + ' to get the asset id.');
+            log.debug('Locating saved asset: ' + stringify(artifact) + ' to get the asset id.');
```
Comment (rushmin) [inline modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js]: Need to check whether debug is enabled.

File: modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js [hunk 2]
```diff
@@ -205,11 +205,11 @@ var module = function () {
...
 
             var id = artifact[0].id || ' ';
 
-            log.info('Setting id of model to ' + id);
+            log.debug('Setting id of model to ' + id);
```
Comment (rushmin) [inline modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js]: Need to check whether debug is enabled.

File: modules/jaggery-apps/app-publisher-web/themes/appm/partials/edit-asset.hbs [hunk 3]
```diff
@@ -268,7 +269,8 @@
                                     <tr>
                                         <th class="col-md-6">URL Pattern</th>
                                         <th class="col-md-2">HTTP Verb</th>
-                                        <th class="col-md-2">Policy Group</th>
+                                        <th class="col-md-2">Policy Group <span
+                                                class="label-required">*</span></th>
```
Comment (rushmin) [inline modules/jaggery-apps/app-publisher-web/themes/appm/partials/edit-asset.hbs]: Formatting issue.

File: modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js (hunk 1 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js]: fixed with commit# https://github.com/lahiru823/carbon-appmgt/commit/70e0e0afb4f357445d16adba3074a1c6301616c5

File: modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js (hunk 2 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-publisher-web/modules/ext/scripts/webapp.action.save.js]: fixed with commit# https://github.com/lahiru823/carbon-appmgt/commit/70e0e0afb4f357445d16adba3074a1c6301616c5

**Pull request merged**